k8s/stacks/observ.cue

```diff
@@ -5,21 +5,22 @@ import (
 	"stakpak.dev/devx/k8s/services/loki"
 	"stakpak.dev/devx/k8s/services/grafana"
 	"stakpak.dev/devx/k8s/services/prometheus"
+	"stakpak.dev/devx/k8s/services/pixie"
 )
 
 ObservabilityStack: v1.#Stack & {
 	$metadata: stack: "ObservabilityStack"
 	components: {
         "loki": loki.#LokiChart & {
 			helm: {
-				version: "6.16.0"
+				version: "2.10.2"
 				release: "loki"
 				values: {}
             }
         }
         "grafana": grafana.#GrafanaChart & {
 			helm: {
-				version: "8.5.1"
+				version: "8.5.11"
 				release: "grafana"
 				values: {}
```

Comment:
this prevents the user from overriding the chart's default values, CUE will complain